scispacy/custom_sentence_segmenter.py

```diff
@@ -26,7 +31,16 @@ def combined_rule_sentence_segmenter(doc: Doc) -> Doc:
 
     @param doc: the spaCy document to be annotated with sentence boundaries
     """
-    segmenter = pysbd.Segmenter(language="en", clean=False)
+    # pysbd doesn't always like short strings
```

Comment:
@danielkingai2 : Can you share few examples of such cases?

Reply:
@nipunsadvilkar the one i noticed was that it turned `.` into ``. And it seemed harmless to not split segments on one single token documents so I just special cased it.

Reply:
I didnt understand it fully. Can you provide an example, please?

Reply:
```
>>> import pysbd
>>> segmenter = pysbd.Segmenter(language='en', clean=False)
>>> segmenter.segment('.')
[]
>>> 
```

Reply:
Got it. Thanks 👍

Reply:
bdw I have fixed https://github.com/nipunsadvilkar/pySBD/issues/27 bug.